index.js

```diff
@@ -277,8 +277,10 @@ if ((os.platform() === 'win32' && require('is-administrator')()) || process.getu
                 interface.address.startsWith('192.168.'));
         })[0];
 
-        if (networkInterface.mac) { config.mac = networkInterface.mac; }
-        if (networkInterface.address) { config.ip = networkInterface.address; }
+        if (networkInterface) {
```

Comment:
This is fine to fix the ref error, but I think we have to look at this more closely in later revisions.  Machines can have multiple NICs, we don't do anything with ipv6 (maybe we don't need to tho), and the IP check is pretty assumptive.  I wonder if this could be a question to the installer, and defaulted to this logic.

Reply:
@darinspivey, you are right! Similar to what we have found on the Node.js Code Library, we weren't capturing IPv6 but in the later revisions we can add this as well to the refactor and improvements.